### ISSUE TITLE
Disconnected foreach-statements within HTML template

### ISSUE THREAD

**Issue by thexpand** [kalessil/phpinspectionsea#40]
The inspections are reporting HTML in a template foreach to be disconnected.
Here is an example:
```
<?php foreach($items as $item): ?>
    <div><?= $item ?></div>
<?php endforeach; ?>
```

In this case, the opening `<div>` and closing `</div>` inside the template foreach are reported as disconnected.

**Comment by kalessil** [comment]
@thexpand : thank you for reporting.

This will be covered by the next release (this [has been fixed recently](https://github.com/kalessil/phpinspectionsea/commit/915287e88d145f95bedd10e699d1380b5124542f))
